### PR TITLE
Use a shorter time for `podman stop`

### PR DESCRIPTION
**Pull request by lukaszachy**
Those are containers to be thrown away, lets stop them much faster. Option isn't configurable.

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

**Comment by lukaszachy** [comment]
Making it draft as it doesn't seem to improve the situation as much as I hoped.  I need to investigate more.

**Comment by psss** [comment]
Just tried this and it works nicely, 9 seconds of waiting saved. Have you encountered some other problem? Shall we include this in `1.30`?

File: tmt/steps/provision/podman.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 DEFAULT_USER = "root"
 DEFAULT_PULL_ATTEMPTS = 5
 DEFAULT_PULL_INTERVAL = 5
+STOP_TIME = "1"  # podman default is 10 seconds
```
Comment (psss) [inline tmt/steps/provision/podman.py:22]: I'd suggest to make this configurable as well. As agreed on the chat, we'll use option groups to put relevant options nicely together in the future. As for now there are not that many options for the `podman` plugin so I'd say adding a few more would not hurt.

**Review by happz — APPROVED** [review]

**Comment by lukaszachy** [comment]
Fixed valid mypy complains....

**Review by psss — APPROVED** [review]
Thanks for the speed-up! Works nicely for me.

**Pull request merged**